### PR TITLE
Add urlencoded header

### PR DESCRIPTION
**Pull request by hayate**

### CODE REVIEW

**Comment by toshipon** [comment]
LGTM

File: src/main/java/com/gengo/client/JsonHttpApi.java [hunk 1]
```diff
@@ -452,7 +452,7 @@ private String httpDelete(HttpURLConnection con) throws GengoException
     private String httpPut(HttpURLConnection con, String query) throws GengoException
     {
         con.setDoOutput(true);
-        con.setRequestProperty("Content-Type", "text/plain");
+        con.setRequestProperty("Content-Type", "application/x-www-form-urlencoded");
```
Comment (ninotoshi) [inline src/main/java/com/gengo/client/JsonHttpApi.java]: Don't you need charset like https://github.com/gengo/gengo-java/blob/master/src/main/java/com/gengo/client/JsonHttpApi.java#L254

**Comment by hayate** [comment]
@ninotoshi HTML 5 seems it does not need it.
Our customer-api definitely is not checking that so it should be OK either way.

**Comment by ninotoshi** [comment]
LGTM

**Pull request merged**